scalanet/src/io/iohk/scalanet/peergroup/udp/StaticUDPPeerGroup.scala

```diff
@@ -225,14 +225,26 @@ class StaticUDPPeerGroup[M] private (
         success
     }
 
+  private def bufferAllocator: RecvByteBufAllocator = {
```

Comment:
couldn't it be `val` or `lazy val` to avoid constant creating of allocator object ?

Reply:
Could be, but it's only ever created once as part of the `lazy val serverBinding`, I just wanted to move out out of there so I can add some comments.

Reply:
Internally it will also just use a single handle instance, unlike `DefaultMaxBytesRecvByteBufAllocator` which does allocate a new one for each message.